server/src/main/java/tabletop/domain/game/wininformation/SingleWinInformation.java

```diff
@@ -1,22 +1,22 @@
 package tabletop.domain.game.wininformation;
 
-import tabletop.domain.user.User;
-
-import java.util.Set;
+import javax.validation.constraints.NotNull;
 
 public class SingleWinInformation {
+    @NotNull
     private Long tournamentId;
-    private User winner;
+    @NotNull
```

Comment:
This should be @NotEmpty. Also currently if you send a JSON without one of these fields you will get a "may not be null" error message with the response. You can change it to @NotEmpty(message = "{win.username}") and in ValidationMessages.properties add this line "win.username = Username cannot be empty!", same for tournamentId. You can use one of the existing lines too